mlflow/models/python_api.py

```diff
@@ -13,6 +14,9 @@
 from mlflow.utils.file_utils import TempDir
 
 _logger = logging.getLogger(__name__)
+UV_INSTALLATION_INSTRUCTIONS = (
+    "Install the 'uv' package by running `pip install uv` or `pip install mlflow[extras]`."
+)
```

Comment:
```suggestion
UV_INSTALLATION_INSTRUCTIONS = (
    "Visit https://docs.astral.sh/uv/getting-started/installation to install uv."
)
```

Can we use https://docs.astral.sh/uv/getting-started/installation instead?

Reply:
We can, but running `curl -LsSf https://astral.sh/uv/install.sh | sh` in MLR is a bit tricky, as you need to manually activate the env like this
<img width="573" alt="image" src="https://github.com/user-attachments/assets/8a9b6ef1-eda9-4026-acc1-68719fbb4fc5">
This also requires users to figure out the platform + which command to use, I think we can add `visit https://docs.astral.sh/uv/getting-started/installation for detailed installation instructions` but providing pip as the easiest installation method for them.

Reply:
Can we use `https://docs.astral.sh/uv/getting-started/installation/#pypi` then?

Reply:
Why not directly use `pip install uv`?

Reply:
If you're using poetry or uv, is `pip install uv` the right way to install it?

Reply:
```suggestion
UV_INSTALLATION_INSTRUCTIONS = (
    "Run `pip install uv` to install uv. See https://docs.astral.sh/uv/getting-started/installation for other installation methods."
)
```

what about this?